src/lib/components/chat-message.svelte

```diff
@@ -20,16 +20,24 @@
 
 	export let timestamp: string | undefined = undefined
 
+	export let onClick: (() => void) | undefined = undefined
+
+	export let leftPadding: number | undefined = undefined
+
 	const isFF = () => {
 		let browserInfo = navigator.userAgent
 		return browserInfo.includes('Firefox')
 	}
 </script>
 
+<!-- svelte-ignore a11y-no-static-element-interactions -->
+<!-- svelte-ignore a11y-click-events-have-key-events -->
```

Comment:
note for self: this sould be resolved and removed

Reply:
I just added a comment for this: `TODO: remove onClick and the ignores once the real design is implemented for babbles`

This is just a temporary hack so that you can open the chat messages, but I think once we have a real design for babbles we will use a different component and then the `onClick` propery can be removed.